web/models.py

```
@@ -1752,6 +1752,65 @@ def display_name(self):
         return self.email.split("@")[0]  # Use part before @ in email
 
 
+class Meetup(models.Model):
+    title = models.CharField(max_length=200)
+    slug = models.SlugField(unique=True, blank=True)
+    description = models.TextField()
+    date = models.DateTimeField()
+    link = models.URLField(blank=True)
+    location = models.CharField(max_length=255, blank=True)
+    event_type = models.CharField(
+        choices=[("online", "Online"), ("in_person", "In Person")], default="online", max_length=10
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)